### PR TITLE
Add option for customizing a delivery method's queue

### PR DESCRIPTION
**Pull request by iheanyi**
By default, `noticed` uses whatever the default queue name is for `ActiveJob` for enqueuing things. At times, the user wants to use different queues for things, so we should be able to customize that queue on a per-delivery method basis when necessary.

### CODE REVIEW

**Comment by excid3** [comment]
Love this, thanks @iheanyi 💪 

Looks like it's just failing on the mimemagic dependencie (what isn't right now? 😅). I think you can just update that with `bundle update && appraisal update` and then it should pass.

File: test/noticed_test.rb [hunk 1]
```diff
@@ -167,4 +171,10 @@ class Noticed::Test < ActiveSupport::TestCase
       With5MinutesDelay.new.deliver(user)
     end
   end
+
+  test 'asserts delivery is queued with different queue' do
+    assert_enqueued_with(queue: 'custom') do
+      WithCustomQueue.new.deliver(user)
```
Comment (excid3) [inline test/noticed_test.rb:177]: I think you need this to be `deliver_later` to fix your test. `deliver` won't enqueue anything cuz it runs immediately.

**Comment by iheanyi** [comment]
@excid3 Welp, tests are failing and I don't know why, haha. So let me go back to the drawing board on this one and see if I can fix it.

File: test/noticed_test.rb (hunk 1 above)
Comment (iheanyi) [reply test/noticed_test.rb:177]: Oh, derp, good catch. Was copying off `delay`, but delay will always be done async.

Comment (iheanyi) [reply test/noticed_test.rb]: Thanks!

**Comment by iheanyi** [comment]
Okay, we're all green now, it looks like. Thanks for your help with this!

**Comment by excid3** [comment]
No problem and thanks for the PR! 💪 

I'll probably cut a new release for it in just a minute.

**Comment by excid3** [comment]
1.3.2 released 🚀

**Pull request merged**